tests/phpunit/tests/rest-api/rest-comments-controller.php

```diff
@@ -119,6 +119,7 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 				array(
 					'comment_content' => "Comment {$i}",
 					'comment_post_ID' => self::$post_id,
+					'status'          => ( rand( 0, 100 ) % 2 === 0 ) ? 'approve' : 'hold',
```

Comment:
```suggestion
					'status'          => ( $i % 2 === 0 ) ? 'approve' : 'hold',
```

To make this a bit more deterministic :-) Randomness in tests is error prone

Reply:
Ha, I liked the randomness, it should always pass... That said, your suggestion is how I had it originally.